apptemplates/__init__.py

```diff
@@ -39,12 +39,15 @@ def get_app_template_dir(app_name):
 
 
 if django.VERSION[:2] >= (1, 9):

```

Comment:
I just noticed we could simplify this line, too! (I know, it's not part of this PR, originally - forgive!)
``` python
if django.VERSION >= (1, 9):
```
Less noise, less distraction. Makes the code more readable.